.github/workflows/check.yml

```diff
@@ -55,8 +55,8 @@ jobs:
 
       - name: test
         run: |
-          zig build test -Denable-tsan=true
-          zig build test -Denable-tsan=true -Dblockstore-db=hashmap -Dfilter=ledger
+          zig build test -Denable-tsan=true --verbose --summary all 
+          zig build test -Denable-tsan=true --verbose --summary all -Dblockstore=rocksdb -Dfilter=ledger
```

Comment:
yes this change is intended. this is necessary to ensure all the same tests are run as previously.

the purpose of these commands as they were already written was to test both the default database and the non-default database (which was hashmapdb). now that rocksdb is the non-default database, it needs to be tested explicitly, instead of hashmapdb.

i don't see how this could be causing problems in ci since it's running the same tests, and when i run it locally it finishes very quickly. but i can't explain what's going on in ci, so i guess anything is possible.